bundles/org.openhab.binding.bmwconnecteddrive/src/main/java/org/openhab/binding/bmwconnecteddrive/internal/dto/charge/Timer.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bmwconnecteddrive.internal.dto.charge;
+
+import java.util.List;
+
+import org.openhab.binding.bmwconnecteddrive.internal.utils.Constants;
+
+/**
+ * The {@link Timer} Data Transfer Object
+ *
+ * @author Bernd Weymann - Initial contribution
+ */
+public class Timer {
+    public String departureTime;// ": "05:00",
+    public boolean timerEnabled;// ": false,
+    public List<String> weekdays;
+
+    /**
+     * "MONDAY",
+     * "TUESDAY",
+     * "WEDNESDAY",
+     * "THURSDAY",
+     * "FRIDAY"
+     * ] '
+     */
+    public String getDays() {
+        if (weekdays == null) {
+            return Constants.UNDEF;
+        }
+        StringBuilder days = new StringBuilder();
+        weekdays.forEach(entry -> {
+            if (days.length() == 0) {
+                days.append(Constants.DAYS.get(entry));
+            } else {
+                days.append(Constants.COMMA).append(Constants.DAYS.get(entry));
+            }
```

Comment:
```suggestion
            if (days.length() != 0){
                days.append(Constants.COMMA);
            }
            days.append(Constants.DAYS.get(entry));
```

Reply:
refactored fixing this issue. See ChargeProfileUtils.formatDays().